services/dummy-test-service/mongodb.go

```diff
@@ -0,0 +1,67 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"go.mongodb.org/mongo-driver/mongo"
```

Comment:
Btw, can you place this mongo driver at the root go.mod and go.mod sum?

Reply:
will check